_source/providenceConfiguration/mainConfiguration/media_processing_top_level.csv

```diff
@@ -0,0 +1,16 @@
+Top level key,Description,Default
+use_external_url_when_available,"If you want original media fetched from URLs to *NOT* be stored in CA, but rather for CA to directly reference the media via the URL used to fetch it set use_external_url_when_available to 1.
```

Comment:
Need to make sure this still works.

Reply:
It is a bit difficult to update the docs and check the quality of the source code, so I decided first documenting and then reviewing the code later.